spec/models/validations/household_validations_spec.rb

```diff
@@ -59,7 +51,7 @@
       end
 
       context "when form year is before 2024" do
-        let(:log_date) { Time.zone.local(2024, 1, 1) }
+        let(:startdate) { Time.zone.local(2024, 1, 1) }
```

Comment:
I removed this, cause I think it got added as part of some other change, but I don't think this would be a problem, because we're building a log instead of creating it and only run the specific validation that's being tested (so it shouldn't fail on date)